vulcanai2/models/dnn.py

```diff
@@ -1,33 +1,40 @@
+# coding=utf-8
+"""Defines the DenseNet class"""
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
-import torch.nn.modules.activation as activations
-import torch.optim as optim
 
 from .basenetwork import BaseNetwork
 from .cnn import ConvNet
```

Comment:
This breaks the imports for ConvNet and DenseNet for some reason

Reply:
the recursive importing is actually messing it up. Do you think is there any better workaround for this?

Reply:
This doc is pretty informative: https://stackoverflow.com/questions/744373/circular-or-cyclic-imports-in-python

Reply:
ok I fixed it. it you replace it from `.cnn import ConvNet` with `from .cnn import *` and the same for dnn

Reply:
would that be ok? it might populate that namespace in the long run but it's a fix for now. @nirtiac any recommendations or is this decent?

Reply:
I think for now we stick with it. We can fix that once we have our `setup.py` ready as we need a __main__ to do a direct import of `vulcanai2`. Does that makes sense?

Reply:
as per the ref from the link you shared

Reply:
I tried running the fashion test file and it can't find it so it doesn't work 100%

Reply:
would you be able to test that @priyatharsan ? before we merge? I just want to make sure it works

Reply:
yeah I will try to fix this tomorrow, unless if @nirtiac has any alternative?